content/blog/2025-04-04-tmil-15/index.md

```diff
@@ -0,0 +1,145 @@
+
+We have also renamed the #gpu channel on Zulip to [#vello](https://xi.zulipchat.com/#narrow/channel/197075-vello), which better reflects the purpose of the channel.
+Progress on the sparse strips renderer has also been continuing at pace:
+
+- [vello#827][]: Added some core API types.
+- [vello#828][]: Imported the implementation of sparse strip production, without rasterisation (Vello Common).
+- [vello#830][], [vello#832][]: Added on CPU rasterisation of sparse strips, with integration tests (Vello CPU).
+- [vello#834][]: Added a tool (Vello Toy) for visualising different stages of the new pipeline.
+- [vello#831][]: Added the fragment shader based GPU rasteriser for sparse strips (Vello Hybrid).
+- [vello#872][]: Enable running Vello Hybrid using WebGL2.
+- [vello#867][]: Benchmarking harness for Vello Common and Vello CPU.
+
+<!-- TODO: Screenshot of WebGL example, in e.g. Firefox on Linux? -->
+
+The current state of the sparse strip renderers is that they can reliably render scenes where all elements are a solid colour.
+<!-- TODO: Mention new mini roadmap? -->
+There are some PRs where you can track ongoing work:
+
+- [vello#878][]: Add clipping to Vello CPU.
+- [vello#883][]: Implements text/glyph rendering.
+- [vello#879][]: Runs Vello Hybrid on Android. <!-- Is now the time to announce Android View? In what form? -->
+
+<!-- Maybe? Screenshot relevant for clipping? -->
+
+You can also follow the progress in [#vello > Vello Hybrid](https://xi.zulipchat.com/#narrow/channel/197075-vello/topic/Vello.20Hybrid) and other threads in [#vello](https://xi.zulipchat.com/#narrow/channel/197075-vello).
+We also are hosting weekly renderer office hours specifically aimed at developing this collaboration.
+Information about how to join these, follow [#office hours](https://xi.zulipchat.com/#narrow/channel/359642-office-hours).
+
+## Parley
+
+Parley is a text layout library.
+It handles text layout, mostly at the level of line breaking and resolving glyph positions.
+The biggest news in March was that an egui community member has opened [a PR][egui#5784] to replace egui's bespoke text handling with Parley, with buy-in from egui's maintainer.
+The author of that change, valadaptive, has been doing fantastic work improving Parley, and has authored many of these changes.
+
+- [parley#306][]: Updated `register_fonts` to take a `Blob`, which allows more efficient memory usage.
+- [parley#312][]: Allows customising how registered fonts are matched, useful for implementing web fonts.
+- [parley#296][]: Makes selected newline characters visible.
+- [parley#316][], [parley#317][]: Provides greater access to the internals of `PlainEditor`.
+- [parley#318][]: `is_done` method on `BreakLines`, for detecting whether linebreaking is finished.
+- [parley#299][]: Fixed text editing for layouts containing inline boxes.
+
+<!-- TODO: Screenshot of egui using Parley? -->
```

Comment:
cc @valadaptive, is there any particularly important change which we can show in a screenshot here (maybe Emoji correctly resolving). I'm willing to collect that screenshot myself, but I don't know what will have changed in a user visible way.

Reply:
Color emoji, RTL, and complex text shaping. I can provide a screenshot in a bit.